lib/r10k/module/local.rb

```diff
@@ -11,10 +11,14 @@ def self.implement?(name, args)
     args.is_a?(Hash) && (args[:local] || args[:type].to_s == 'local')
   end
 
-  def version
+  def self.statically_defined_version(name, args)
```

Comment:
Since these args aren't used, should we drop them?

Reply:
No, we call this with args, since the other module types use them in their implementations: https://github.com/puppetlabs/r10k/blob/main/lib/r10k/module/definition.rb#L11

Reply:
In that case, it should be just `*`.

Reply:
Do you want to make that edit?